src/components/Reports/GDPR/SubscriberGdpr.js

```diff
@@ -22,20 +22,34 @@ const SubscriberGdpr = ({ location, dependencies: { dopplerApiClient } }) => {
       setState({ loading: true });
       const email = replaceSpaceWithSigns(extractParameter(location, queryString.parse, 'email'));
       const responseSubscriber = await dopplerApiClient.getSubscriber(email);
-      if (responseSubscriber.success) {
+      const allFields = await dopplerApiClient.getUserFields();
+      if (responseSubscriber.success && allFields.success) {
```

Comment:
how do you know when one fails who is it? or it doesnt matter?

Reply:
Here we need both, what do you suggest?

Reply:
but you show the same message for both cases?

Reply:
Yes. Else it has no meaning to show incomplete information.